### PR TITLE
[core]: repos and runcounter.txt are stored inside coreWorkingDir

### PR DESCRIPTION
**Pull request by miltalex**
Closes OCTRL-129.

### CODE REVIEW

File: core/confsys/service.go [hunk 1]
```diff
@@ -129,19 +125,8 @@ func (s *Service) NewRunNumber() (runNumber uint32, err error) {
...
 		var rnf string
-		rnf, err = s.src.Get("o2/control/run_number_file")
-		if err != nil {
-			return
-		}
-		rnf, err = homedir.Expand(rnf) // Resolve ~ into home dir
-		if err != nil {
-			return
-		}
+		rnf = viper.GetString("coreWorkingDir") + "runcounter.txt"
```
Comment (teo) [inline core/confsys/service.go]: Are we certain that `coreWorkingDir` has a trailing slash?
Generally it shouldn't: the default shouldn't have a trailing `/`, and if the user passes a `/path/with/trailing/slash/`, it should be sanitized to remove it.

Then, this line becomes
```
		rnf = viper.GetString("coreWorkingDir") + "/runcounter.txt"
```

Comment (teo) [reply core/confsys/service.go]: Addendum: all user-provided paths must be sanitized to ensure the absence of a trailing slash.

Comment (miltalex) [reply core/confsys/service.go]: The `coreWorkingDir` is sanitized [here](https://github.com/AliceO2Group/Control/blob/master/core/config.go#L184).  I followed the same pattern used for the `repositoriesPath`.

File: common/utils/utils.go [hunk 2]
```diff
@@ -53,3 +53,9 @@ func EnsureTrailingSlash(path *string) {
 
 // helper func to package strings up nicely for protobuf
 func ProtoString(s string) *string { return &s }
+
+func RemoveTrailingSlash(path *string) {
```
Comment (teo) [inline common/utils/utils.go]: To be checked if there is an `os.path` function that can already do this for us.

Comment (miltalex) [reply common/utils/utils.go]: Change it to `filepath.Clean()`

**Pull request merged**